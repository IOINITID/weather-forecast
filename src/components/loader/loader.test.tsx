import React from 'react';
import renderer from 'react-test-renderer';
import Loader from '../loader/loader';

describe(`Loader component`, () => {
  it(`Loader component should render correctly`, () => {
    const tree = renderer.create(<Loader />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
