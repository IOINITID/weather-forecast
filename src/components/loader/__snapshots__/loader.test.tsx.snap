// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader component Loader component should render correctly 1`] = `
<SvgrURL
  className="loader"
  height="100"
  width="100"
/>
`;
